XML/XSD/Castable.xsd

```diff
@@ -27,6 +27,23 @@
     </xs:documentation>
   </xs:annotation>
 
+  <!-- Castable type enumeration
+       Can be expanded for future use -->
+  <xs:simpleType name="CastableType">
```

Comment:
I don't think having `Type` at all is the correct solve. Type is irrelevant and handled by `Book`

Reply:
`Type` exists currently

Reply:
i see that, but i'm still failing to see the usage, even in that scenario. It's not defined as anything currently, just required.